platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -72,11 +72,13 @@
  * @param platformPublisher                        configuration for the platform publisher scheduler
  * @param consensusEventStream                     configuration for the consensus event stream scheduler
  * @param roundDurabilityBuffer                    configuration for the round durability buffer scheduler
- * @param statusStateMachine                       configuration for the status state machine scheduler
- * @param platformStatusNexus                      configuration for the status nexus scheduler
  * @param signedStateSentinel                      configuration for the signed state sentinel scheduler
  * @param signedStateSentinelHeartbeatPeriod       the frequency that heartbeats should be sent to the signed state
```

Comment:
merge conflict error here. The word "sentinel" was deleted from this doc

Reply:
fixed